dev-requirements-py3.txt

```diff
@@ -1,5 +1,5 @@
 flake8==3.7.8
-demisto-sdk>=0.3.2
+demisto-sdk>=0.3.3
```

Comment:
what is that?

Reply:
content build issue, which is fixed with new SDK